src/beebop_deploy.py

```diff
@@ -169,23 +168,12 @@ def redis_configure(container, cfg):
 
 
 def api_configure(container, cfg):
-    if db_is_initialised(container, cfg):
-        print("[api] Storage db already exists, doing nothing")
-    else:
-        print("[api] Downloading storage database")
-        args = ["./scripts/download_databases"]
-        mounts = [docker.types.Mount("/beebop/storage",
-                  cfg.volumes["storage"])]
-        container.client.containers.run(str(cfg.api_ref), args, mounts=mounts,
-                                        remove=True)
-
-
-def db_is_initialised(container, cfg):
-    res = container.exec_run(["stat",
-                              os.path.join("/beebop/storage",
-                                           cfg.api_db_name)])
-    return res[0] == 0
-
+    print("[api] Downloading storage database")
+    args = ["./scripts/download_databases"]
+    mounts = [docker.types.Mount("/beebop/storage",
+                cfg.volumes["storage"])]
+    container.client.containers.run(str(cfg.api_ref), args, mounts=mounts,
+                                    remove=True)
```

Comment:
Doesn't this mean that it's going to download databases on every redeploy? That might be time consuming or not something that you want, considering that databases will be updated infrequently compared to how often redeploys might happen. Sounds like refreshing dbs could be a flag on deploy command?

Reply:
Ah ok, cool had forgotten that!